src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,16 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Lottery();
+        System.out.println("firstBall "
+                + firstBall.getRandomBall());
+
+        Lottery secondBall = new Lottery();
+        System.out.println("secondBall "
+                + secondBall.getRandomBall());
+
+        Lottery thirdBall = new Lottery();
+        System.out.println("thirdBall "
+                + thirdBall.getRandomBall());
```

Comment:
Let's create all the balls and print them within the loop. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class).